ports/atmel-samd/boards/pewpew_m4/mpconfigboard.mk

```diff
@@ -48,7 +48,7 @@ CIRCUITPY_KEYPAD_SHIFTREGISTERKEYS = 0
 CIRCUITPY_KEYPAD_KEYMATRIX = 0
 CIRCUITPY_MATH = 1
 CIRCUITPY_STAGE = 1
-CIRCUITPY_SYNTHIO = 1
+CIRCUITPY_SYNTHIO = 0
```

Comment:
@deshipu Do you use synthio?

Reply:
The new `synthio.Synthesizer` is very unlikely to fit here, but I could shuffle things around so that the new features are optional: `synthio.Synthesizer` `Envelope` and `Note` all raise NotImplemented, OR put them in a fresh module (but you have to tell me the module's name)

Reply:
This is fine for now. I'll leave it up to @deshipu how they want to get it back. (The new features sound so good!)